test/cut/test_padding_cut.py

```diff
@@ -220,6 +220,14 @@ def test_pad_simple_cut(libri_cut):
     np.testing.assert_almost_equal(pre_mixed_feats, mixed_feats[:1604, :])
 
 
+def test_pad_cut_no_features(libri_cut):
+    libri_cut.features = None
+    padded = libri_cut.pad(desired_duration=20.0)
+
+    # TODO: implement padding for cut without features in libri_cut.pad()
+    assert padded.duration <= 20.0
```

Comment:
It should be `==`, right?

Reply:
Because padding hasn't been implemented for raw audio, the duration here should be less than 20.0.
It will be implemented in future (in next PR, I think).

Reply:
I remove this unit test because we assumes that `Cut.features` is not None when calling `Cut.pad()`.